### DISCUSSION TITLE
blog/hermit-programmers-are-dead

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#39] [Announcements]
# blog/hermit-programmers-are-dead

With the advent of cloud computing and AI, there is no room for hermit programmers. You better diversify your skills or perish in the process, so don’t call yourself just a programmer anymore.

https://www.cesarsotovalero.net/blog/hermit-programmers-are-dead.html

**Comment by alvarezbatet** [comment]
I've heard saying independent programmers will dominate the AI landscape as it becomes more open source. Institutions need to follow very strict rutines which makes their progress slow, meanwhile there's millions of independent programmers around the world that will be able to catch up with new discoveries and deploy their projects faster an cheaper than big buisinesses.

**Comment by cesarsotovalero** [comment]
Hi @alvarezbatet, there is no doubt that a few very talented programmers have sparked some of the most radical innovation engines through their open-source contributions and entrepreneurship (think of Linus Torvalds, or more recently Vitalik Buterin). The reason, as you mention, is that individuals can certainly take more risks than big tech companies. However, the current AI turnover is not about mind power but a matter of machine power and resources. I mean, computational resources: millions of GPUs! ...running somewhere in Iceland or Greenland, where the energy to keep them running is cheaper. The reason is that training AI models, such as LLMs and GPT-based models, requires immense amounts of energy. So much energy that even entire governments cannot afford it. Yes, [the papers may be public](https://arxiv.org/abs/1706.03762), and even the models are open-source (such as the [llama](https://github.com/facebookresearch/llama) model from Meta) but the code is not the point. I think the current AI ~~bubble~~ landscape is similar to other previous episodes. It has happened with operating systems (Microsoft), search engines (Google), and social media (Facebook), where the market had a single big winner. For AI, only time will tell.
